tests/integration-tests/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import server from "../../src/index";
+
+afterAll(async () => {
+  // workaround. It'd fails to kill server before jest finish its execution
+  // so it results in failure of jest to stop.
+  setTimeout(() => {
+    server.close();
+  }, 1000);
+});
+
+describe("index.ts", () => {
+  test('requests the "/" route - should return expected response', async () => {
+    const expectedResponse = "hello world with fastify";
+    const res = await server.inject({
+      method: "GET",
+      url: "/",
+    });
```

Comment:
as mentioned in: https://www.fastify.io/docs/latest/Guides/Testing built-in `.inject` function is pretty neat! I didn't need any extra dependency or anything to set up because of this. I already like this library😄 
We might need to set up util or third-party dependency when we extend our tests more deeper, but for now, fastly is capable of taking care of integration test